tests/common/factories/annotation.py

```diff
@@ -151,3 +154,34 @@ def timestamps(self, create, extracted, **kwargs):  # pylint:disable=unused-argu
         # pylint:disable=attribute-defined-outside-init
         self.created = self.created or datetime.datetime.now()
         self.updated = self.updated or datetime.datetime.now()
+
+    @factory.post_generation
+    def slim(self, create, extracted, **kwargs):  # pylint:disable=unused-argument
+        if not create or not kwargs.get("new"):
```

Comment:
This is opt-int, the whole test suite is very lax with integrity around groups, users. 

We can use it in more tests if we move into making the new table more useful.